packages/@guardian/source-foundations/src/colour/palette.tsx

```diff
@@ -92,13 +92,6 @@ const colors = {
 	],
 } as const;
 
-/**
- * [Storybook](https://guardian.github.io/source/?path=/docs/packages-source-foundations-palette--page) •
- * [Design System](https://theguardian.design/2a1e5182b/p/492a30-light-palette)
- *
- * Colour palette organised by type e.g. 'error', or pillar e.g. 'opinion'
- */
-
```

Comment:
if we remove this we'll lose vscode's intellisense tips for palette for consumers:

<img width="596" alt="image" src="https://user-images.githubusercontent.com/867233/180973657-d97330c9-df08-429a-8e60-40bff7fc5fc8.png">

Reply:
Reverted this change.